modules/data.atmosphere/R/met2CF.ALMA.R

```diff
@@ -102,7 +102,7 @@ met2CF.PalEON <- function(in.path, in.prefix, outfolder, start_date, end_date, l
           met[[v]] <- ncvar_get(nc = nc1, varid = v)
         } else {
           tmp      <- ncvar_get(nc = nc1, varid = v)
-          met[[v]] <- abind(met[[v]], tmp)
+          met[[v]] <- abind::abind(met[[v]], tmp)
```

Comment:
@mdietze just checking, just importing abind from abind function to use here correct?  That is 

"Combine multi-dimensional arrays. This is a generalization of cbind and rbind. Takes a sequence
of vectors, matrices, or arrays and produces a single array of the same or higher dimension."